source/layouts/layout.erb

```diff
@@ -1,176 +1,168 @@
-                <image src="/assets/images/govuk-logotype-crown.png" class="govuk-header__logotype-crown-fallback-image"></image>
+                <image src="/assets/images/govuk-logotype-crown.png" alt="GOV.UK Crown Logotype" class="govuk-header__logotype-crown-fallback-image"></image>
               </svg>
               <span class="govuk-header__logotype-text">
                 GOV.UK
               </span>
             </span>
-            <span class="govuk-header__product-name">
+        <span class="govuk-header__product-name">
               Pay
             </span>
-          </a>
-        </div>
-        <div class="govuk-header__content">
-          <button type="button" role="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
-          <nav>
-            <ul id="navigation" class="govuk-header__navigation" aria-label="Top Level Navigation">
-            <%
-              {
-                'Features' => '/using-govuk-pay/',
-                'Get started' => '/getstarted/',
-                'Documentation' => 'https://docs.payments.service.gov.uk',
-                'Support' => '/support/',
```

Comment:
`/support` and `/getstarted` currently redirect to `/support/` etc - unless we are removing these redirects we might as well keep the trailing slashes and save the user a few milliseconds?

Reply:
this is outdated